Description: Set TLSv1.2 as minimum TLS version. LP: #1856428
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1856428


Index: nss-3.48-1ubuntu1/nss/lib/ssl/sslsock.c
===================================================================
--- nss-3.48-1ubuntu1.orig/nss/lib/ssl/sslsock.c
+++ nss-3.48-1ubuntu1/nss/lib/ssl/sslsock.c
@@ -96,7 +96,7 @@ static sslOptions ssl_defaults = {
  * default range of enabled SSL/TLS protocols
  */
 static SSLVersionRange versions_defaults_stream = {
-    SSL_LIBRARY_VERSION_TLS_1_0,
+    SSL_LIBRARY_VERSION_TLS_1_2,
     SSL_LIBRARY_VERSION_TLS_1_3
 };
 
